src/System.IO.Abstractions.TestingHelpers/System.IO.Abstractions.TestingHelpers.csproj

```diff
@@ -3,7 +3,7 @@
         <AssemblyName>System.IO.Abstractions.TestingHelpers</AssemblyName>
         <RootNamespace>System.IO.Abstractions.TestingHelpers</RootNamespace>
         <Description>A set of pre-built mocks to help when testing file system interactions.</Description>
-        <TargetFrameworks>net5.0;netstandard2.1;netstandard2.0;net461</TargetFrameworks>
+        <TargetFrameworks>net6.0;net5.0;netstandard2.1;netstandard2.0;net461</TargetFrameworks>
```

Comment:
OMG, I should actually have done this in https://github.com/TestableIO/System.IO.Abstractions/pull/777/files 😬 Thanks!